gusto/timeloop.py

```diff
@@ -345,12 +346,12 @@ def __init__(self, equation, scheme, io, spatial_methods=None,
         else:
             self.physics_schemes = []
 
-        for _, phys_scheme in self.physics_schemes:
+        for parametrisation, phys_scheme in self.physics_schemes:
             # check that the supplied schemes for physics are explicit
-            assert isinstance(phys_scheme, ExplicitTimeDiscretisation), \
-                "Only explicit time discretisations can be used for physics"
+            # assert isinstance(phys_scheme, ExplicitTimeDiscretisation), \
```

Comment:
This check needs to become a check for whether the time discretisation is explicit and whether this is allowed for the physics scheme (add a logical to the physics scheme).